client/src/app/index.js

```diff
@@ -39,6 +40,11 @@ class App extends Component {
 
   authFun = async () => {
     try {
+      const {
+        location: { pathname, search },
+      } = window;
+      ReactGa.initialize('UA-168097204-1');
+      ReactGa.pageview(pathname + search);
```

Comment:
Since this is inside the `try` block, any exception here would trigger the auth error handling on line 59.  That might be fine, but I just wanted to point it out.